newIDE/app/src/ObjectEditor/ObjectsEditorService.js

```diff
@@ -1,3 +1,4 @@
+// @flow
```

Comment:
I added some typing here to avoid mistakes.
Is there other files related to object editors where flow should be added?

Reply:
Nothing in mind, that's good already!